chroma-manager/tests/integration/core/remote_operations.py

```diff
@@ -372,7 +372,38 @@ def _ssh_address(self, address, command, expected_return_code=0, timeout=TEST_TI
         logger.info("SSH address = %s, args = %s" % (address, args))
 
         # Create ssh connection
-        ssh.connect(address, **args)
+        try:
+            ssh.connect(address, **args)
+        except paramiko.ssh_exception.SSHException, e:
+            def print_result(r):
+                return "rc: %s\n\nstdout:\n%s\n\nstderr:\n%s" % \
+                       (r.rc, r.stdout, r.stderr)
+
+            ping_result1 = Shell.run(['ping', '-c', '1', '-W', '1', address])
+            ifconfig_result = Shell.run(['ifconfig', '-a'])
+            ip_route_ls_result = Shell.run(['ip', 'route', 'ls'])
+            ping_gw_result = Shell.run(['ping', '-c', '1', '-W', '1', '10.14.80.1'])
```

Comment:
I thought about that.  For the amount of work it would be I didn't think it was worth it given that this debug is pretty limited to our own infrastructure and hopefully not that long lived.